README.md

```diff
@@ -25,37 +21,62 @@ Build Wasm and native code:
 cargo build --release
 ```
 
-## Run
+Great! You have already compile the Centrifuge Chain!
 
-### Tests
+## Tests
 
-```bash
-cargo test -p centrifuge-runtime --release
-```
+There are two kinds of tests, one related to how the *Centrifuge Chain* works itself
+and another one to verify how it works in a more real environment as a parachain.
 
-### Start local Relay chain(alice and bob) and Parachain(alice)
+### Chain tests
 
-Prerequisites:
-- [docker](https://docs.docker.com/get-docker/)
-- [*jd*](https://stedolan.github.io/jq/)
+The following command will run unitary and integration tests:
 
-Start relay chain
 ```bash
-./scripts/init.sh start-relay-chain
-```
-
-Start  centrifuge-chain as parachain
-```bash
-./scripts/init.sh start-parachain
+cargo test -p centrifuge-runtime --release
 ```
 
-Note: the command above will show logs and block until the parachain is stopped
-Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1`.
-
-Onboard parachain to Relay chain
-```bash
-./scripts/init.sh onboard-parachain
-```
+### Environment tests
+
+You can deploy a relay chain and connect a Centrifuge Chain node as parachain
+to it to verify how it behaves in the entire environment (end-to-end).
+
+0. Prerequisites. You must install these tools before:
+    - [docker](https://docs.docker.com/get-docker/)
+    - [*jd*](https://stedolan.github.io/jq/)
+
+1. Start a local [relay chain](https://wiki.polkadot.network/docs/learn-architecture#relay-chain).
+It contains two [validator](https://wiki.polkadot.network/docs/learn-validator) nodes
+    (Alice and Bob):
+    ```bash
+    ./scripts/init.sh start-relay-chain
+    ```
+    After a few seconds you can see the block production of the relay chain using the [polkadot.js (on localhost:9944)](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2Flocalhost%3A9944#/explorer) client.
+
+    *Note: You can stop the relay chain using `./scripts/init.sh stop-relay-chain`*
+
+2. Start a *Centrifuge Chain* as [parachain](https://wiki.polkadot.network/docs/learn-parachains).
+It run a [collator](https://wiki.polkadot.network/docs/learn-collator) node:
+    ```bash
+    ./scripts/init.sh start-parachain
+    ```
+    *Note: the command above will show logs and block until the parachain is stopped.
+    Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1`.
+    If you had a previous state, you can reset the node using `purge` after the command.*
+
+    Similar to the relay chain, you can explore the *Centrifuge Chain* using the [polkadot.js (on localhost:11946)](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2Flocalhost%3A11946#/explorer) client.
+    You will see the block production frozen until you connect it to the relay chain.
+
+3. Onboard parachain to Relay chain (it will connect the *Centrifuge Chain* to the relay chain).
```

Comment:
```suggestion
3. Onboard the parachain
    This step will have the targeted parachain onboarded in the relay chain. The parachain will NOT produce blocks until this step is completed successfully.
```

Note that it can the Altair chain or the development chain, not necessarily just the `Centrifuge` chain